src/components/tx/SecurityWarnings/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { Alert, Checkbox, FormControlLabel, SvgIcon, Typography } from '@mui/material'
+import type { AlertProps } from '@mui/material'
+
+import { SecuritySeverity } from '@/services/security/modules/types'
+import AlertIcon from '@/public/images/notifications/alert.svg'
+
+import css from './styles.module.css'
+
+export const _mapSeverity = (severity: SecuritySeverity): NonNullable<AlertProps['severity']> => {
+  switch (severity) {
+    case SecuritySeverity.HIGH:
+      return 'error'
+
+    case SecuritySeverity.LOW:
+      return 'warning'
+
+    case SecuritySeverity.NONE:
+      return 'success'
+  }
+}
+
+export const _mapRisk = (severity: SecuritySeverity): string => {
+  switch (severity) {
+    case SecuritySeverity.HIGH:
+      return 'Critical risk'
+
+    case SecuritySeverity.LOW:
+      return 'Low risk'
+
+    case SecuritySeverity.NONE:
+      return 'No issues found'
+  }
+}
```

Comment:
Can you please make enums or records out of this? Too much syntax for such a simple mapping.